src/test/libcephfs/test.cc

```diff
@@ -655,6 +655,21 @@ TEST(LibCephFS, Xattrs_ll) {
   ceph_shutdown(cmount);
 }
 
+TEST(LibCephFS, check_dummy_op) {
```

Comment:
document the test purpose

Reply:
document in this code itself? or where exactly?

Reply:
in the test

Reply:
Are you sure I add it here? As none of the existing tests in this file have their purposes documented. Although I do have it done in `libcephfs.h`

Reply:
It's good practice. Unfortunately not everyone has done so.

Reply:
Got it. I'll document it.

Reply:
added. thanks.